### PR TITLE
Adding Jobs and Leads plugin

### PR DESCRIPTION
**Pull request by torrinworx**
Adding v1 of Jobs and Leads plugin

### CODE REVIEW

File: plugins/jobs_and_leads/server.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+const dummyBusinessContactInfo = [
```
Comment (Nefsen402) [inline plugins/jobs_and_leads/server.js:1]: Are these all fake? I don't want to be adding real businesses here...

Comment (torrinworx) [reply plugins/jobs_and_leads/server.js:1]: It's just dummy data from ChatGPT to match the Google Places api response, it wasn't based off of any real businesses.

Comment (Nefsen402) [reply plugins/jobs_and_leads/server.js:1]: Okay.

**Comment by Nefsen402** [comment]
Looks good.

**Pull request merged**